### DISCUSSION TITLE
Siembol presentation at Black Hat Europe, Nov 10-11

### DISCUSSION THREAD

**Discussion by c-rindi** [G-Research/siembol#276] [Events]
Marian Novotny will be presenting Siembol at Black Hat Europe in London, likely in-person this time. :smile:

[Conference](https://www.blackhat.com/eu-21/arsenal/schedule/index.html#siembol-an-open-source-real-time-siem-tool-based-on-big-data-technologies-25165) dates for [Black Hat Europe Tools Arsenal](https://www.blackhat.com/eu-21/) are Nov. 10-11 - all times listed are GMT.

Wednesday, November 10 | 12:00pm-1:00pm ( Virtual )
Wednesday, November 10 | 2:15pm-3:30pm ( Business Hall, Arsenal Station 3 )
Thursday, November 11 | 1:00pm-1:50pm ( Business Hall, Arsenal Station 1 ) 

https://www.blackhat.com/eu-21/

Watch past presentations on the [GR-OSS YouTube channel](https://www.youtube.com/playlist?list=PLNWn4-IA6Bx5fEpWIZPiAfimR9QuSLcfz).

**Comment by tabathad** [comment]
Miss the Seimbol BlackHat EU 2021 presentation? You can watch the replay on the [GR-OSS YouTube channel](https://www.youtube.com/channel/UCI3GL6q9RUL3swr3HftS8Qg), [take a look](https://youtu.be/eIMKlV0dGl4)!
